src/components/atoms/button/styles.css

```diff
@@ -0,0 +1,8 @@
+.button {
+  background-color: var(--color-azure-blue);
```

Comment:
For better re-usability I think it would be better to define a color palette in this format:
```
--color-primary
--color-primary-dark
--color-primary-light

--color-secondary
--color-secondary-dark
--color-secondary-light

--color-error

--color-black
--color-white

--color-gray-default
--color-gray-dark
--color-gray-light

....
```